services/workflows-service/src/common/filters/AllExceptions.filter.ts

```diff
@@ -34,49 +35,45 @@ export class AllExceptionsFilter extends BaseExceptionFilter {
       .json({
         errorCode: String(HttpStatusCode[serverError.getStatus()]),
         message: serverError.message,
+        errors,
         statusCode: serverError.getStatus(),
         timestamp: new Date().toISOString(),
         path: request.url,
       });
   }
 
   private getHttpException(exception: unknown) {
-    let serverError;
     if (this.isHttpException(exception)) {
       // Nest http error handler
-      serverError = exception;
-    } else {
-      serverError = new InternalServerErrorException({
-        cause: inspect(exception),
-      });
+      return exception;
     }
-    return serverError;
+
+    return new InternalServerErrorException({
+      cause: inspect(exception),
+    });
   }
 
   isHttpException(error: unknown): error is HttpException {
-    return (
-      isObject(error) &&
-      'status' in error &&
-      typeof (error as HttpException).getStatus() === 'number'
-    );
+    return error instanceof HttpException;
```

Comment:
People also reported an error due to `typeof (error as HttpException).getStatus() === 'number'` since non HttpException errors don't have `getStatus`.